src/amp/components/Epic.tsx

```diff
@@ -192,14 +315,55 @@ const buildUrl = (
 	)}`;
 };
 
+const getReminderDate = (date: Date = new Date()): Date => {
+	const monthsAhead = date.getDate() < 20 ? 1 : 2;
+	date.setMonth(date.getMonth() + monthsAhead);
+
+	return date;
```

Comment:
I think maybe we should return a new `Date` object instead of mutating the one passed in?

Reply:
Yeah, I had a similar thought. I copied this straight from the non-AMP implementation, but I'll refactor this one.